src/main/webapp/dev/js/main/instructorFeedbackEdit.js

```diff
@@ -535,15 +535,19 @@ function enableNewQuestion() {
 
     moveAssignWeightsCheckbox($newQuestionTable.find(`#rubricAssignWeights-${NEW_QUESTION}`));
 
-    if ($(`#generateOptionsCheckbox-${NEW_QUESTION}`).prop('checked')) {
```

Comment:
Is generateOptionsCheckbox an id in any other type of questions?

Reply:
Because amongst your changes it doesn't seem like you changed any jsp/html file. So it could either break something or it might be that the initial id was a typo.

Reply:
The initial id `generateOptionsCheckbox` was a typo which was the reason why adding question again after canceling didn't trigger the check. Doing a project-wide search, `generateOptionsCheckbox` is currently only used in 3 places and all those are part of `instructorHelpSession.jsp` ( I am assuming its outdated code)

Reply:
@tshradheya LGTM